sdk/cognitiveservices/azure-cognitiveservices-language-luis/azure/cognitiveservices/language/luis/authoring/tests/test_examples.py

```diff
@@ -0,0 +1,166 @@
+from azure.cognitiveservices.language.luis.authoring import LUISAuthoringClient
+from baseTest import *
+import unittest
+
+
+class ExamplesTests(unittest.TestCase):
+
+    @use_client
+    def test_list_examples(self, client: LUISAuthoringClient):
+            examples = client.examples.list(global_app_id, global_version_id)
+            assert(len(examples) != 0)
+
+    @use_client
+    def test_list_examples_for_empty_application_returns_empty(self, client: LUISAuthoringClient):
+            app_id = client.apps.add({
+                'name': "Examples Tests App",
+                'description': "New LUIS App",
+                'culture': "en-us",
+                'domain': "Comics",
+                'usageScenario': "IoT"
+            })
+            examples = client.examples.list(app_id, global_version_id)
+            client.apps.delete(app_id)
+            assert(len(examples) == 0)
```

Comment:
Depending on how the Swagger is implemented list operations might return an iterator, and not a list. If you get an iterator, doing "len" on it would be a problem.
The recommendation would be to convert explicitly to a list, then if it's already a list it's a no-op, and if not it changes it from an iterator to a list and you 100% your type is consistent:
```python
examples = list(client.examples.list(app_id, global_version_id)) # Cast as a list in case it's an iterator
assert(len(examples) == 0)  # You 100% sure now "len" is applied correctly
```

Reply:
Good catch! will fix it.